tools/RAiDER/cli/__main__.py

```diff
@@ -29,18 +29,9 @@ def main():
     
     sys.argv = [args.process, *unknowns]    
 
-    if args.process == 'calcDelays':
-        from RAiDER.cli.raider import calcDelays
-    elif args.process == 'downloadGNSS':
-        from RAiDER.cli.raider import downloadGNSS
-    elif args.process == 'calcDelaysGUNW':
-        from RAiDER.cli.raider import calcDelaysGUNW
-    else:
-        raise NotImplementedError(f'Process {args.process} has not been fully implemented')
-
```

Comment:
All these imports are done dynamically in the try block below, so these are unessary.

Likewise, argparse already ensures `args.process` is one of the implemented processes.